2377/2/versus/versus.rb

```diff
@@ -0,0 +1,164 @@
+# rubocop:disable Metrics/LineLength
```

Comment:
We've added HoundCI to make u write code according to style guides.  Don't disable it!